lib/IRGen/MetadataLayout.h

```diff
@@ -222,6 +225,10 @@ class ClassMetadataLayout : public NominalMetadataLayout {
 
   Offset getFieldOffsetVectorOffset(IRGenFunction &IGF) const;
 
+  unsigned getNumImmediateMembers() const {
+    return NumImmediateMembers;
+  }
```

Comment:
What actually uses this and what does it mean?

Reply:
Instead of storing the size of instantiated class metadata in the template, I'm computing it as the size of the superclass metadata plus the number of immediate members. This part isn't fully baked yet and I'm not going to land it until I have more stuff fleshed out